### DISCUSSION TITLE
no php found error

### DISCUSSION THREAD

**Discussion by annuanitha** [pimcore/pimcore#9345] [Q&A / Support]
I was trying to install a pimcore skeleton project version2.8.13 .
when i tried to install using vendor\bin\pimcore-install cmd
i got a error no php found try to install or add the path in env variable.
even though the path is already included in env variable I am still facing the same error.

I am using a windows 10 machine.
The php version is 7.3.26

**Comment by brusch** [comment]
Pimcore X requires PHP 8 at least. 
Windows doesn't support Shebang, you have to run the command like this: `php vendor/bin/pimcore-install`. Please keep in mind that Pimcore is not officially supported on Win.

**Reply by annuanitha** [reply]
I am not talking about pimcoreX.. 
I am trying to install Pimcore 6.8.7. 
I am not able to install it since i am getting php not found error.

In env variable I have added the path of php7

**Comment by ComputerMasta** [comment]
3. COMPONENTS

Bitnami Pimcore Stack ships with the following software versions:

  - Pimcore 6.9.3
  - Apache 2.4.46
  - MySQL 8.0.24
  - PHP 7.4.16
  - phpMyAdmin 5.1.0

4. REQUIREMENTS

To install Bitnami Pimcore Stack you will need:

    - Intel x86 or compatible processor
    - Minimum of 256 MB RAM
    - Minimum of 150 MB hard drive space
    - TCP/IP protocol support
    - Compatible operantig systems:
      - An x86 or x64 Linux operating system.
      - A Windows operating system such as Windows Vista, Windows 7, Windows 8, Windows 10, Windows Server 2008 or Windows Server 2012.
			- An OS X operating system.
[Check this out if still need help](https://factory.hr/pimcore-knowledge-base).
